engine/movie/title.asm

```diff
@@ -201,7 +198,7 @@ _TitleScreen:
 	ld a, -112
 	ldh [hWY], a
 
-	ld a, $1
+	ld a, 1
```

Comment:
`ld a, TRUE` since it looks like `[hCGBPalUpdate]` is read as a Boolean in `VBlank3`.